### PR TITLE
Make FOR DISTRIBUTION optional when there is only one distribution

### PR DESCRIPTION
**Pull request by visill**

### CODE REVIEW

File: qdb/ops/ops.go [hunk 1]
```diff
@@ -19,6 +19,20 @@ func CreateKeyRangeWithChecks(ctx context.Context, qdb qdb.QDB, keyRange *kr.Key
...
+	if keyRange.Distribution == "default" {
+		list, err := qdb.ListDistributions(ctx)
+		if err != nil {
+			return spqrerror.New(spqrerror.SPQR_NO_DISTRIBUTION, "error while selecting list of distributions")
+		}
+		if len(list) == 0 {
+			return spqrerror.New(spqrerror.SPQR_NO_DISTRIBUTION, "you didn't have any distributions")
+		}
+		if len(list) != 1 {
+			return spqrerror.New(spqrerror.SPQR_NO_DISTRIBUTION, "distributions count not equal one, use DISTRIBUTION BY syntax")
```
Comment (EinKrebs) [inline qdb/ops/ops.go:31]:
```suggestion
			return spqrerror.New(spqrerror.SPQR_NO_DISTRIBUTION, "distributions count not equal one, use FOR DISTRIBUTION syntax")
```

File: test/regress/tests/console/expected/add.out [hunk 2]
```diff
@@ -5,19 +5,23 @@
...
 
+CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1;
+ERROR:  you didn't have any distributions.
 CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
     add distribution    
 ------------------------
  distribution id -> ds1
 (1 row)
 
-CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1 FOR DISTRIBUTION ds1;
```
Comment (EinKrebs) [inline test/regress/tests/console/expected/add.out:14]: Consider adding new tests rather than rewriting existing

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: test/regress/tests/console/expected/add.out (hunk 2 above)
Comment (reshke) [reply test/regress/tests/console/expected/add.out:14]: 💯

Comment (visill) [reply test/regress/tests/console/expected/add.out:14]: I changed the logic of creating the key range. We have a test testing the logic of creating key range and distribution. I'm changing it without doing a duplicate test. If you think you need to do a separate test, then I will do a separate key_range test.

**Pull request closed without merging**